libs/ledger-live-common/src/families/solana/api/chain/index.ts

```diff
@@ -82,6 +83,11 @@ export type ChainAPI = Readonly<{
   config: Config;
 }>;
 
+// Naive mode, allow us to filter in sentry all this error comming from Sol RPC node
+const remapErrors = (e) => {
+  throw new NetworkDown(e);
```

Comment:
Maybe `NetworkError` would be more relevant? (we created it recently)

Reply:
I don't see NetworkError on code base. Can you provide me the file ?

Reply:
also `new NetworkDown(e)` won't make the error correctly mapped, it probably would be better to `new NetworkError(e.message)`

if we introduce such new NetworkError, don't forget to add to the exclusion list of sentry too ( https://ledgerhq.atlassian.net/wiki/spaces/WALLETCO/pages/3757539598/How+to+ignore+an+error+to+be+sent+to+Sentry )